src/components/student/exam/questionsSet/QuestionsList.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="html">
+  <table class="table is-striped">
+    <tr
+      v-for="question in questions"
+      class="is-primary is-clickable">
+        <td
+          @click="selectQuestion"
+          :class="{'is-active': isActive(question.id)}"
+          :id="question.id"
+          :key="question.id">
+          {{question.question}}
+        </td>
+    </tr>
+  </table>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+export default {
+  computed: {
+    ...mapGetters({
+      questions: 'activeExam/questions'
+    })
+  },
+
+  data () {
+    return {
+      question: 1
```

Comment:
It's a bad idea to make have the question represented by an integer id in the component, while at the same time make it an object with an id in the store.